keps/prod-readiness/sig-node/4727.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 4580
+alpha:
+  approver: "@haircommander"
```

Comment:
this should be a PRR approver https://github.com/kubernetes/community/blob/master/sig-architecture/production-readiness.md#submitting-a-kep-for-production-readiness-approval

Reply:
Thanks for your guidance